testing/iframe.js

```diff
@@ -64,14 +66,14 @@ export function createFixtureIframe(fixture, initialIframeHeight, opt_beforeLoad
   return new Promise((resolve, reject) => {
     // Counts the supported custom events.
     const events = {
-      'amp:attached': 0,
-      'amp:bind:initialize': 0,
-      'amp:bind:setState': 0,
-      'amp:bind:rescan-template': 0,
-      'amp:error': 0,
       'amp:form-service:initialize': 0,
-      'amp:load:start': 0,
-      'amp:stubbed': 0,
+      [AmpEvents.ATTACHED]: 0,
+      [BindEvents.INITIALIZE]: 0,
+      [BindEvents.SET_STATE]: 0,
+      [BindEvents.RESCAN_TEMPLATE]: 0,
+      [AmpEvents.ERROR]: 0,
+      [AmpEvents.LOAD_START]: 0,
+      [AmpEvents.STUBBED]: 0,
```

Comment:
It's a computed property name, a feature of ES6. http://es6-features.org/#ComputedPropertyNames